app/js/functions.js

```diff
@@ -1,7 +1,7 @@
 // DATASHEET CONFIG
 var DEFAULT_DATA = "data/nst_2011.csv";
 var DEFAULT_TOPO = "data/us-states.topojson";
-var USER_DIRECTORY = "uploader/upload/";
+var USER_DIRECTORY = "../../server/uploader/upload/";
```

Comment:
This can be switched to /server/uploader/upload/ as that will point to the root of the project instead.